test/parallel/test-debugger-pid.js

```diff
@@ -29,8 +29,8 @@ interfacer.on('line', function(line) {
     switch (++lineCount) {
       case 1:
         line = line.replace(/^(debug> *)+/, '');
-        var msg = 'There was an internal error in Node\'s debugger. ' +
-                  'Please report this bug.';
+        const msg = 'There was an internal error in Node\'s debugger.' +
+                    'Please report this bug.';
```

Comment:
@jasnell, I have ran './configure && make -j8 test', './node ./test/parallel/test-debugger-pid.js' and 'make test' but I did not get error. Which test should I run for CI